tornado_rest_client/test/test_api.py

```diff
@@ -165,6 +175,19 @@ def test_replace_path_tokens(self):
         self.assertEquals(str(ret), '/test/abcd/info')
 
     @testing.gen_test
+    def test_new_style_path_with_property_access(self):
+        test_consumer = RestConsumerTest(client=RestClientTest())
+
+        # the path with an arg must exist as an access method, and fail if you
+        # don't supply the arg.
+        with self.assertRaises(TypeError):
```

Comment:
This isn't me setting the type itself .. instead, its a verification that the attribute `test_consumer.test_path_with_new_access_type_with_res` is indeed a property, not a `callable` method. This is an actual error being returned by Python, not me.

If I remove the `with self.assert...`:
```
======================================================================
ERROR: test_new_style_path_with_property_access (tornado_rest_client.test.test_api.TestRestConsumer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/diranged/git/nextdoor/kingpin/.venv/lib/python2.7/site-packages/tornado/testing.py", line 136, in __call__
    result = self.orig_method(*args, **kwargs)
  File "/Users/diranged/git/nextdoor/kingpin/.venv/lib/python2.7/site-packages/tornado/testing.py", line 529, in post_coroutine
    timeout=timeout)
  File "/Users/diranged/git/nextdoor/kingpin/.venv/lib/python2.7/site-packages/tornado/ioloop.py", line 457, in run_sync
    return future_cell[0].result()
  File "/Users/diranged/git/nextdoor/kingpin/.venv/lib/python2.7/site-packages/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "/Users/diranged/git/nextdoor/kingpin/.venv/lib/python2.7/site-packages/tornado/gen.py", line 270, in wrapper
    result = func(*args, **kwargs)
  File "/Users/diranged/git/nextdoor/kingpin/.venv/lib/python2.7/site-packages/tornado/testing.py", line 512, in pre_coroutine
    result = f(self, *args, **kwargs)
  File "/Users/diranged/git/nextdoor/tornado_rest_client/tornado_rest_client/test/test_api.py", line 183, in test_new_style_path_with_property_access
    test_consumer.test_path_with_new_access_type_with_res()
TypeError: 'RestConsumerTest' object is not callable
```